src/objective-c/GRPCClient/private/GRPCHost.m

```diff
@@ -129,7 +130,8 @@ - (nullable grpc_call *)unmanagedCallWithPath:(NSString *)path
     }
     channel = _channel;
   }
-  return [channel unmanagedCallWithPath:path completionQueue:queue];
+  NSString *name = serverName ? serverName : _address;
```

Comment:
Let's make gRPC core take care of this case, and simply pass `NULL` to `grpc_channel_create_call`.